packages/talos_agent/src/agent/state_manager.rs

```diff
@@ -576,7 +583,7 @@ mod tests {
 
         tx_metrics
             .expect_send()
-            .withf(move |param_event| expect_event_at_time(param_event, EventName::CandidateReceivedByTalos, decided_at))
+            .withf(move |param_event| expect_event_at_time(param_event, EventName::CandidateReceivedByTalos, candidate_received_at))
```

Comment:
Probably better to call `CandidateReceivedByTalos` as `CandidateReceivedByCertifier` as agent is also Talos :)